addons/sale_timesheet/models/project_update.py

```diff
@@ -1,62 +1,68 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from collections import defaultdict
+
 from odoo import api, models
 from odoo.tools import float_utils, format_amount, formatLang
 
-from collections import defaultdict
-
 class ProjectUpdate(models.Model):
     _inherit = 'project.update'
 
     @api.model
     def _get_template_values(self, project):
+        template_values = super(ProjectUpdate, self)._get_template_values(project)
+        services = self._get_services_values(project)
+        profitability = self._get_profitability_values(project)
+        show_sold = template_values['project'].allow_billable and len(services.get('data', [])) > 0
+        show_profitability = profitability
```

Comment:
The `show_profitability` will not a boolean value since we give a `dict`. If you want a boolean value in this variable, you should do this:
```suggestion
        show_profitability = bool(profitability)
```
🙂 

Or I miss something ? 🤔